synergy-core/fedora/35/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN dnf clean all && dnf -y update && dnf -y install \
     qt5-qttools-devel \
     libcurl-devel \
     openssl-devel \
+    desktop-file-utils \
```

Comment:
Without this instruction I can build this image but with this instruction I have the following error: 
```
#5 0.345 0 files removed                                                                                                                                                                                    
#5 18.40 Fedora 35 - aarch64                             4.2 MB/s |  75 MB     00:17                                                                                                                        
#5 27.90 Fedora 35 openh264 (From Cisco) - aarch64       4.9 kB/s | 2.5 kB     00:00    
#5 29.30 Fedora Modular 35 - aarch64                     2.7 MB/s | 3.2 MB     00:01    
#5 31.70 Fedora 35 - aarch64 - Updates                   1.5 MB/s | 2.9 MB     00:01    
#5 31.70 Errors during downloading metadata for repository 'updates':
#5 31.70   - Downloading successful, but checksum doesn't match. Calculated: 7f2396bfa1e386818c018ab37fd607c6ccfec05796477f149fbf5b6c11323802(sha256)  Expected: 93e6c59a46b4f72d276d2a468d909a48f81ab20f63d3a0836f8e93fa1a0ed91c(sha256) 
#5 31.70   - Zchunk header checksum didn't match expected checksum
#5 31.70   - Curl error (60): SSL peer certificate or SSH remote key was not OK for https://mirror.yandex.ru/fedora/linux/updates/35/Everything/aarch64/repodata/repomd.xml [SSL: no alternative certificate subject name matches target host name 'mirror.yandex.ru']
#5 31.70   - Curl error (60): SSL peer certificate or SSH remote key was not OK for https://mirror.yandex.ru/fedora/linux/updates/35/Everything/aarch64/repodata/93e6c59a46b4f72d276d2a468d909a48f81ab20f63d3a0836f8e93fa1a0ed91c-comps-Everything.aarch64.xml.gz [SSL: no alternative certificate subject name matches target host name 'mirror.yandex.ru']
#5 31.70   - Downloading successful, but checksum doesn't match. Calculated: 0fa665d3bee64128ee584250fbfcaeee221d3b4eed37f74daf8e6461e85ff6f8efd2cede1744aafe3a78dca697739ca6d35fb7d085680fa64d4b4a3f83715616(sha512) 0fa665d3bee64128ee584250fbfcaeee221d3b4eed37f74daf8e6461e85ff6f8efd2cede1744aafe3a78dca697739ca6d35fb7d085680fa64d4b4a3f83715616(sha512) 0fa665d3bee64128ee584250fbfcaeee221d3b4eed37f74daf8e6461e85ff6f8efd2cede1744aafe3a78dca697739ca6d35fb7d085680fa64d4b4a3f83715616(sha512)  Expected: 008270a92982d1ccdad6a6eef2389b3156356b3719f95f8ed2e436b6fa75356584f5d0f00632c47cfe61db950633d077d2d2ee5cb10988faf5f476dd1943ffd7(sha512) bcec91025dc6c0e17d98a9f99da36140eb796d9cb180ee8e96526b7cf47a2f853cd917e81a3baf4a865e8bd9de0914755a3682ac46195cc9f498eb9aefd39398(sha512) ab1bb2f91ef5fc471cfef1c091419d823c76ac310ce59eb2e2d7431b339c082a952c4dd4952d8d5d048b777ff4df6982b115a987df4b5a2b464cf9e6cba8393a(sha512) 
#5 31.70   - Curl error (60): SSL peer certificate or SSH remote key was not OK for https://mirror.yandex.ru/fedora/linux/updates/35/Everything/aarch64/repodata/7ba1e902746b8c7a101de0f3839a7d0ec39347be28afb76ecdee920b94f4d253-updateinfo.xml.zck [SSL: no alternative certificate subject name matches target host name 'mirror.yandex.ru']
#5 31.70   - Curl error (60): SSL peer certificate or SSH remote key was not OK for https://mirror.yandex.ru/fedora/linux/updates/35/Everything/aarch64/repodata/65c3e7bf4a2b4c8edc9056e102bed1ac26686ce68ac86f27f878105071f9a01e-prestodelta.xml.zck [SSL: no alternative certificate subject name matches target host name 'mirror.yandex.ru']
#5 31.71 Error: Failed to download metadata for repo 'updates': Checksum error /var/cache/dnf/updates-3a0989866655ad58/repodata/dfe682c0f900db488a3227534b01af25a7ae92d4a093000dc7f68a764892105c-primary.xml.zck: Unable to read zchunk lead
------
executor failed running [/bin/sh -c dnf clean all && dnf -y update && dnf -y install     avahi-compat-libdns_sd-devel     avahi-compat-libdns_sd     cmake3     git     gpg     make     libXtst-devel     qt5-qtbase-devel     qt5-qtdeclarative-devel     qt5-qttools-devel     libcurl-devel     openssl-devel     desktop-file-utils     rpm-build     rpmlint     libnotify-devel     glib2-devel     gdk-pixbuf2-devel     libxkbfile-devel     && rm -rf /var/cache/yum]: exit code: 1
```